source/administration/incident-response-application.rst

```diff
@@ -0,0 +1,466 @@
+Incident Response Application (EE, Closed Alpha)
+------------------------------------------------------
+
+The incident response application is available in closed Alpha, in `Enterprise Edition <https://mattermost.com/pricing/>`_. Supported in Mattermost 5.12 and later.
+
+// TODO: Add a screenshot or a video
+
```

Comment:
Add an intro on what workflow apps are in one sentence, .e.g "Connect all your workflows, automate repetitive tasks and collaborate on incidents within one secure platform."